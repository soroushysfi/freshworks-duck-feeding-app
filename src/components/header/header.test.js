import { shallow } from 'enzyme'
import { createMemoryHistory } from "history";


import HeaderComponent from './header.component'

describe("Header Component tests", () => {
it('expect to match HeaderComponent snapshot', () => {
    expect(shallow(<HeaderComponent />)).toMatchSnapshot()

})

it('expect route to be at homepage', () => {
    const history = createMemoryHistory();
    expect(history.location.pathname).toBe("/");

})

})


