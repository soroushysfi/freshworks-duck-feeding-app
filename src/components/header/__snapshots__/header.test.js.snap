// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component tests expect to match HeaderComponent snapshot 1`] = `
<div
  className="link-div"
>
  <Link
    className="top-link-style"
    to="/"
  >
    <span
      className="link-text"
    >
      Form
    </span>
  </Link>
  <Link
    className="top-link-style"
    to="/report"
  >
    <span
      className="link-text"
    >
      Report (last 10 dates)
    </span>
  </Link>
</div>
`;
